bucket/ani-cli.json

```
@@ -1,20 +1,21 @@
 {
-    "version": "3.0",
+    "version": "3.3",
     "description": "A CLI to browse and watch anime. This tool scrapes the site gogoanime.",
     "homepage": "https://github.com/pystardust/ani-cli",
     "license": "GPL-3.0-or-later",
```

```suggestion
    "license": "GPL-3.0-or-later",
    "suggest": {
        "git": "git"
    },
```

Suggest Git Bash requirement, in line with the other shell scripts in this bucket.